sdk/samples/pong_app.c

```diff
@@ -0,0 +1,331 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#define _GNU_SOURCE
+#include <assert.h>
+#include <bsd/string.h>
+#include <getopt.h>
+#include <linux/limits.h>
+#include <signal.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <time.h>
+#include <unistd.h>
+#include "mesh_dp.h"
+#include <pthread.h>
+#include <stdatomic.h>
+#include <sched.h>
+#include "pingpong_common.h"
+
+#define DEFAULT_RECV_IP "127.0.0.1"
+#define DEFAULT_RECV_PORT "10001"
+#define DEFAULT_SEND_IP "127.0.0.1"
+#define DEFAULT_SEND_PORT "9001"
```

Comment:
Move those define to *common.h and unify them between ping/pong

Reply:
I moved IPs, but ports have to be different in each app so I left them